packages/farcaster-js-hub-rest/src/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './hubRestApiClient.js';
 export * as FarcasterEpochTimestamp from './farcasterEpochTimestamp.js';
 export * from './logger.js';
 export * from './openapi/index.js';
+export * from './utils.js';
```

Comment:
Do you foresee a developer having use for importing the functions in this file?

Reply:
nope, removed.